### ISSUE TITLE
Confirmed cases and deaths not updating since 2021-04-18

### ISSUE THREAD

**Issue by epa** [OxCGRT/covid-policy-tracker#86]
In the big CSV file of all data at https://github.com/OxCGRT/covid-policy-tracker/raw/master/data/OxCGRT_latest.csv there are rows for 2021-04-19 and later dates, but these rows are blank in the latest rows.  For example the line beginning `United States,USA,,,NAT_TOTAL,20210418` has values `31670031,567217` for confirmed cases and confirmed deaths, but the following line (beginning `United States,USA,,,NAT_TOTAL,20210419`) has empty for these columns.  This happens for all countries.  Could you check please?

**Comment by Troy8898** [comment]
Yes, I have observed the same issue for the CSV file of all data. Is there a different file for confirmed cases and deaths for 2021-04-19 and later dates?

**Comment by saptahash** [comment]
Hi, thanks for raising this. We'll have a look at this now!

**Comment by glorioso** [comment]
Thanks yeah- same issue for me.

**Comment by tboby** [comment]
This should now be resolved.

**Comment by Troy8898** [comment]
Thank you for fixing the data feed! It is working for me now.
